components/modal/modal.css

```diff
@@ -0,0 +1,35 @@
+.modal {
+  /* Hidden by default */
+  display: none;
+  position: fixed; /* Stay in place */
+  z-index: 9999; /* Sit on top */
+  top: 0;
+  left: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0, 0, 0); /* Fallback color */
+  background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
+}
```

Comment:
Could not preview modal. It does not opens on clicking. need to fix it.

Reply:
I forgot to link modal js. Fixed the issue now!